### ISSUE TITLE
[v3.0.0-pre1] InvalidOperationException: EditForm requires either a Model parameter, or an EditContext parameter, please provide one of these.

### ISSUE THREAD

**Issue by hakenr** [havit/Havit.Blazor#345]
After upgrading GOR-G3, the ProjectList page fails with
```
System.InvalidOperationException: EditForm requires either a Model parameter, or an EditContext parameter, please provide one of these.
   at Microsoft.AspNetCore.Components.Forms.EditForm.OnParametersSet()
   at Microsoft.AspNetCore.Components.ComponentBase.CallOnParametersSetAsync()
   at Microsoft.AspNetCore.Components.ComponentBase.RunInitAndSetParametersAsync()
```

With the same DB, only the HavitBlazor3 branch fails (just the new package + replacement of HxInputCheckbox+Switch with HxCheckbox+Switch). The master branch does not throw the exception.

**Comment by hakenr** [comment]
The console windows also contains this exception at the same time:
```
Microsoft.AspNetCore.Components.WebAssembly.Rendering.WebAssemblyRenderer[100]
      Unhandled exception rendering component: Object reference not set to an instance of an object.
System.NullReferenceException: Object reference not set to an instance of an object.
   at Havit.Blazor.Components.Web.Bootstrap.HxSearchBox`1.<DisposeAsyncCore>d__169[[Havit.GoranG3.Contracts.Projects.ProjectReferenceVM, Havit.GoranG3.Contracts, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null]].MoveNext()
   at Havit.Blazor.Components.Web.Bootstrap.HxSearchBox`1.<DisposeAsync>d__168[[Havit.GoranG3.Contracts.Projects.ProjectReferenceVM, Havit.GoranG3.Contracts, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null]].MoveNext()
```

**Comment by hakenr** [comment]
resolved (also resolves #344)
